services/user.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+User = get_user_model()
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = "",
+        first_name: str = "",
+        last_name: str = ""
+) -> QuerySet:
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name,
+    )
+    return user
+
+
+def get_user(user_id: int) -> User | None:
+    try:
+        user = User.objects.get(id=user_id)
+        return user
+    except User.DoesNotExist:
+        return None
```

Comment:
Use https://docs.djangoproject.com/en/4.1/topics/http/shortcuts/#get-object-or-404
try-except is so bad in this case